index.js

```diff
@@ -0,0 +1,73 @@
+const testLineup = ['1B', '2B', '3B', 'C', 'OF', 'OF', 'OF', 'P', 'SS']
```

Comment:
manually sorted array is hard to maintain. maybe can do this in next line

testLineup = testLineup.sort()

also should rename variable to make it more clear its already sorted.